awx/ui/client/src/workflow-results/workflow-results.block.less

```diff
@@ -37,6 +37,12 @@
     }
 }
 
+.WorkflowResults-leftSide{
+    .card{
+        height:86vh;
```

Comment:
this set's the left side as a percentage of the view height, so it will only look correct at a specific browser window height.  If you resize your browser window you should be able to reproduce this.

The actual change would be something along the lines of 

```
height:calc(~"100vh - XYZpx");
```

where XYZ is the height the top bar and padding take up.